src/content/app/genome-browser/state/browser-sidebar-modal/browserSidebarModalSlice.ts

```diff
@@ -29,7 +29,7 @@ export enum BrowserSidebarModalView {
   SEARCH = 'search',
   BOOKMARKS = 'Previously viewed',
   SHARE = 'Share',
-  DOWNLOADS = 'Downloads',
+  DOWNLOADS = 'Download',
```

Comment:
If you are also changing the values in enums, then I wonder whether we should also change the name of the enum key as well (`DOWNLOAD` instead of `DOWNLOADS`). If yes, then we would do so in all apps that have such enums (genome browser, entity viewer, species page). If we make this change, it would be purely for our (developers') benefit; so it depends on how much more annoying the form "downloads" is than "download".